cli/src/upload.rs

```diff
@@ -17,8 +17,6 @@ use bundle::{
 use codeowners::CodeOwners;
 use constants::{EXIT_FAILURE, EXIT_SUCCESS};
 #[cfg(target_os = "macos")]
-use context::junit::junit_path::JunitReportStatus;
```

Comment:
This was reported from `cargo check` which `trunk check` doesn't support yet and is not reported by Clippy